src/ui/pages/article/BaseViewArticlePage.js

```
@@ -0,0 +1,19 @@
+import { BasePage } from '../BasePage';
+import { ArticleContentBlock } from '../../components/articles/ArticleContentBlock';
+
+export class BaseViewArticlePage extends BasePage {
+  constructor(page, userId = 0) {
+    super(page, userId);
+    this.article = new ArticleContentBlock(this.page, userId);
+  }
+
+  async open(url) {
+    await this.step(`Open article page: ${url}`, async () => {
```

Optional: validate the `url` parameter (or throw a clear error) when `open(url)` is called with an invalid/falsy value, so callers fail fast with a helpful message instead of attempting to navigate to an incorrect URL.